started/goroutines.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Using Goroutines
+---
+
+Since Fyne v2.6.0 all Fyne events and callbacks happen on the same goroutine - this allows
+for higher performance and much better thread safety than any previous version.
+
+With this change applications that wish to use goroutines (to do background updates for example)
+have a little more work to do to get the same safety and performance as internal widgets.
+
+## Updating from a goroutine
+
+Any time your app includes a goroutine (using the `go` keyword) to invoke graphical updates from
```

Comment:
I think this is a bit wordy, how about:

"Any time your app invokes Fyne APIs from a goroutine your code created, you should use the `fyne.Do`...